diff --git a/nautilus_open_any_terminal/nautilus_open_any_terminal.py b/nautilus_open_any_terminal/nautilus_open_any_terminal.py
index 0000000..1111111 100644
--- a/nautilus_open_any_terminal/nautilus_open_any_terminal.py
+++ b/nautilus_open_any_terminal/nautilus_open_any_terminal.py
@@ -43,7 +43,11 @@
 _ = gettext
-for localedir in [expanduser("~/.local/share/locale"), "/usr/share/locale"]:
+for localedir in [
+    expanduser("~/.local/share/locale"),
+    "/run/current-system/sw/share/locale",
+    "/usr/share/locale",
+]:
     try:
         trans = translation("nautilus-open-any-terminal", localedir)
